src/components/message/message.test.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable react/jsx-boolean-value */
+import * as React from "react";
+import { useState } from "react";
+import Button from "../button";
+import Message from "./message.component";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+
+import {
+  messagePreview,
+  messageChildren,
+  messageTitle,
+  messageDismissIcon,
+} from "../../../cypress/locators/message/index";
+
+const testData = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+
+const MessageComponent = ({ ...props }) => {
+  const [isOpen, setIsOpen] = useState(true);
+  return (
+    <div>
+      {!isOpen && <Button onClick={() => setIsOpen(true)}>Open Message</Button>}
+      <Message open={isOpen} onDismiss={() => setIsOpen(false)} {...props}>
+        Some custom message
+      </Message>
+      <span type="warning"> </span>
+    </div>
+  );
+};
+
+context("Tests for Message component", () => {
+  describe("should check Message component properties", () => {
+    it.each(["error", "info", "success", "warning"])(
+      "should check %s variant for Message component",
+      (variant) => {
+        CypressMountWithProviders(<MessageComponent variant={variant} />);
+        messagePreview().should(
+          "have.css",
+          "background-color",
+          "rgb(255, 255, 255)"
```

Comment:
there is no point fot this test, we are checking the same background for each variant